ds3-autogen-net/src/main/java/com/spectralogic/ds3autogen/net/NetHelper.java

```diff
@@ -0,0 +1,75 @@
+package com.spectralogic.ds3autogen.net;
+
+import com.google.common.collect.ImmutableList;
+import com.spectralogic.ds3autogen.api.models.Arguments;
+
+import java.util.stream.Collectors;
+
+import static com.spectralogic.ds3autogen.utils.Helper.capFirst;
+import static com.spectralogic.ds3autogen.utils.Helper.sortConstructorArgs;
+import static com.spectralogic.ds3autogen.utils.Helper.uncapFirst;
+
+public final class NetHelper {
+    private NetHelper() {
+        // pass
+    }
+
+    public static String constructor(final ImmutableList<Arguments> args) {
+        return sortConstructorArgs(args)
+                .stream()
+                .map(i -> getType(i) + " " + uncapFirst(i.getName()))
+                .collect(Collectors.joining(", "));
+    }
+
+    /**
+     * Gets the type of an argument, converting the type from Contract type to Java type.
```

Comment:
Why is the NetHelper converting from Contract type to Java type? Is this a typo?